lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -25,7 +25,7 @@ export class PublicClientApplication extends ClientApplication implements IPubli
 
     // Active requests
     private activeSilentTokenRequests: Map<string, Promise<AuthenticationResult>>;
-    private atsAsyncMeasurement?: InProgressPerformanceEvent = undefined;
+    private astsAsyncMeasurement?: InProgressPerformanceEvent = undefined;
```

Comment:
This should be ok (non breaking) since this is private and local to msal-browser